doc-content/enterprise-only/openshift/openshift-usage-ref.adoc

```diff
@@ -0,0 +1,84 @@
+[id='openshift-usage-ref']
+= OpenShift usage quick reference
+
+To deploy, monitor, manage, and undeploy {PRODUCT} templates on {OPENSHIFT}, you can use the OpenShift Web console or the `oc` command. 
+
+For instructions about using the Web console, see https://access.redhat.com/documentation/en-us/openshift_container_platform/3.11/html-single/getting_started/#getting-started-developers-console[Create and build an image using the Web console].
+
+For detailed instructions about using the `oc` command, see https://access.redhat.com/documentation/en-us/openshift_container_platform/3.11/html/cli_reference/[CLI Reference]. The following commands are likely to be required:
+
+* To create a project, use the command:
```

Comment:
Search all "use the command:" instances in this section and replace with "use this command:"., per IBM SG (use complete sentences preceding a colon).

Reply:
went with "the following command" as I use this formula elsewhere in the docs. I'm making a lost of things where we need to work out guidelines and this will be there.